queuelistener/listener.go

```diff
@@ -196,6 +201,15 @@ func Listen(o Options) (net.Listener, error) {
 		return nil, err
 	}
 
+	if o.ProxyProtocol {
+		log.Printf("Proxy protocol listener")
```

Comment:
Don't use `log` package. Use `o.Log` element. Check out comment for this element in the `Options` struct.
https://github.com/bojleros/skipper/blob/d08410aab25c9ac71a5ada46ae38bfcf5128a46e/queuelistener/listener.go#L71-L72